WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlow.swift

```diff
@@ -22,7 +22,10 @@ class BloggingRemindersFlow {
         let flowStartViewController = makeStartViewController(for: blog, tracker: tracker, source: source)
         let navigationController = BloggingRemindersNavigationController(
             rootViewController: flowStartViewController,
-            onDismiss: onDismiss)
+            onDismiss: {
+                NoticesDispatch.unlock()
+                onDismiss?()
+            })
```

Comment:
This diff was in conflict with `develop`. I chose this resolution, i.e. applied https://github.com/wordpress-mobile/WordPress-iOS/pull/16835, because that PR has a bug fix added later on in the flow and it was appropriate to prefer it (which is what the rest of the merge did anyway).

Reply:
I agree.